syntribos/issue.py

```diff
@@ -42,7 +42,7 @@ class Issue(object):
 
     def __init__(self, defect_type, severity, description, confidence,
                  request=None, response=None, impacted_parameter=None,
-                 init_signals=[], test_signals=[], diff_signals=[]):
+                 init_signals=[], test_signals=[], diff_signals=[], failed_strings=[]):
```

Comment:
Please set the default to `None` and check if it is not `None` then initialise the array. It is a bad idea to have `[]` as a default args.

Reply:
Done 👍🏼